src/OnboardingSPA/components/AdminBar/index.js

```diff
@@ -1,7 +1,40 @@
+import { __ } from '@wordpress/i18n';
+import { Icon, wordpress } from '@wordpress/icons';
+import { useSelect } from '@wordpress/data';
+import { store as nfdOnboardingStore } from '../../../OnboardingSPA/store';
+
 const AdminBar = () => {
+	const { currentUserInfo } = useSelect( ( select ) => {
+		return {
+			currentUserInfo:
+				select( nfdOnboardingStore ).getCurrentUserDetails(),
+		};
+	}, [] );
+
+	if ( ! currentUserInfo ) {
+		return null;
+	}
+
 	return (
 		<div className="nfd-onboarding-header__admin-bar">
-			Admin Bar Goes Here
+			<div className="nfd-onboarding-header__admin-bar__wplogo">
+				<Icon icon={ wordpress } />
+				<span>{ __( 'WordPress', 'wp-module-onboarding' ) }</span>
+			</div>
+			<div className="nfd-onboarding-header__admin-bar__profile">
+				<span className="nfd-onboarding-header__admin-bar__profile__greeting">
+					<span>
+						{ __( 'Howdy!', 'wp-module-onboarding' ) },{ ' ' }
```

Comment:
Do we have an exclamation mark? Also, let's not use empty strings like that. Can't it be part of the same string?

Reply:
yes there is an exclamation, will remove the space